src/user.ts

```diff
@@ -956,7 +960,7 @@ export class User {
   }
 
   private async sendModifiedPasswordEmail(user: SlUserDoc, req): Promise<void> {
-    if (this.config.local.sendPasswordChangedEmail) {
+    if (this.config.local.sendPasswordChangedEmail && !this.config.mailer.useCustomMailer) {
```

Comment:
Here is no event emited

Reply:
The event is emitted here:

https://github.com/perfood/couch-auth/blob/325148330d65f4950ba46ec08b637e07eff25f4d/src/user.ts#L959